### ISSUE TITLE
Task: translate hallucinations

### ISSUE THREAD

**Issue by joseph2mi** [guillaumekln/faster-whisper#354]
I noticed that when I translate compared to when I transcribe the same language and audio file, I get a lot more hallucinations and repeated loops. Has anyone figured out how to solve that error? My parameters are below:

segments, info = model.transcribe(
    "/media/joseph2mi/Shared Dual/VAHF/VAHF Videos/"+srt_file_name+".mp4.",
    beam_size=10, 
    vad_filter=True,
    vad_parameters=dict(min_silence_duration_ms=50),
    language = "en",
    max_initial_timestamp = 3.0,
    condition_on_previous_text = True,
    length_penalty = 1.0,
    temperature = 0,
)

**Comment by phineas-pta** [comment]
r u trying to translate FROM english ?

**Comment by joseph2mi** [comment]
No, my original transcription is in Vietnamese, but since faster-whisper doesn't have a task translate, I just say the language is "en" and it translates it from Vietnanese to English.

**Comment by phineas-pta** [comment]
it does have `task="translate"`

**Comment by joseph2mi** [comment]
Regardless, it doesn't solve my issue. When I put it on task = translate, it still has hallucinations and repeated loops that doesn't occur when I just transcribe it

**Comment by phineas-pta** [comment]
did u specify `language="vi"`

**Comment by joseph2mi** [comment]
i did, there is nothing different between the audio and paramaters except that one is transcribing in vietnamese and one is translating the vietnamese into english. but from my observations on many audio, "task = translate" is much more prone to hallucinations where transcribing is not for some reason

**Comment by guillaumekln** [comment]
Have you tried leaving most parameters to their default value? For example:

```python
model.transcribe("file.mp4", language="vi", task="translate")
```

**Comment by joseph2mi** [comment]
Hi, the whole reason why I added all these different parameters in with different experimentations was because the default parameters would end up with massive hallucinations. I managed to minimize the hallucinations by increasing temperature, decreasing no speech threshold, and decreasing compression ratio threshold, but my main thing I'm curious about (research wise) is why the task translate function seems to be less reliable then transcribing the same audio.

some of my observations are that increasing the memory usage (float 32) and beam size helps mitigate hallucinations a little bit, but if it is just translating, i don't understand why these hallucinations would only occur when translating.

**Comment by archive-r** [comment]
Do you get the same results with openai/whisper rather than faster-whisper? If so, it's probably an issue with the model itself, which is hard to fix here.
